src/esp/sim/Simulator.cpp

```diff
@@ -178,10 +192,21 @@ void Simulator::reset() {
   if (physicsManager_ != nullptr)
     physicsManager_
         ->reset();  // TODO: this does nothing yet... desired reset behavior?
+
+  for (int iAgent = 0; iAgent < agents_.size(); ++iAgent) {
+    auto& agent = agents_[iAgent];
+    agent::AgentState::ptr state = agent::AgentState::create();
+    sampleRandomAgentState(state);
```

Comment:
This will now conflict with reset behavior in python as of #435 (reset to initial vs. random).

Reply:
Fixed.